Docs/source/Software/Apps.rst

```diff
@@ -1,3 +1,170 @@
 ************
 Application
-************
\ No newline at end of file
+************
+
+
+Command Line Interface (CLI)
+============================
+
+This interface is meant to be used for debugging and monitoring the BPS. 
+A list of all valid CLI commands is included below.
+
+"Help"
+######
+
+| Displays a help menu with all available commands.
+| **Argument(s)**: N/A
+
+"Voltage"
+#########
+
+| Displays voltage levels (both module-specific and total voltage) in Volts and the safety of the battery.
+| **Argument(s)**: 
+- **none**: prints every module's voltage
+- **module [module #]**: prints specific module's voltage
+- **total**: prints total battery pack's voltage
+- **safety**: prints safety status
+
+"Current"
+#########
+
+| Displays current readings as measured by the high- and low-precision current sensors in Amps.
+| **Argument(s)**: 
+- **none**: prints high- and low- precision current data
```

Comment:
I would change this since we do not have low and high precision readings anymore. We can change it in the code later.